test/integration/forms/form.js

```diff
@@ -483,7 +483,31 @@ context('Patient Action Form', function() {
         return fx;
       })
       .routeForm(_.identity, '11111')
-      .routeFormDefinition()
+      .routeFormDefinition(fx => {
+        fx.components.push(
+          {
+            label: 'Survey',
+            tableView: false,
+            questions: [
+              {
+                label: '',
+                value: '',
+                tooltip: '',
+              },
+            ],
+            values: [
+              {
+                label: '',
+                value: '',
+                tooltip: '',
+              },
+            ],
+            key: 'fields.survey',
+            type: 'survey',
```

Comment:
An empty survey component's value is `{}` but it will error if the field value is `[]`